sonar-java-plugin/src/test/java/org/sonar/plugins/java/JavaSensorTest.java

```diff
@@ -277,6 +277,15 @@ void does_not_insert_SymbolicExecutionVisitor() throws IOException {
         "ParameterReassignedToCheck"
       );
   }
+  
+  @Test
+  void info_log_when_no_SE_rules_enabled() throws IOException {
+    Loggers.get(PerformanceMeasure.class).setLevel(LoggerLevel.INFO);
```

Comment:
`INFO` is the default, no need to set it explicitly. (And in cases where we do want to set it explicitly, why not use `logTester.setLevel()`?)